rmd/Menus.json

```diff
@@ -1140,9 +1140,9 @@
     "Enabled": true
   },
   "20806": {
-    "OriginalText": "マルチプレイ中は\nリーダー似外はアクセスできません。\n\n[a 03]※ストーリーを進めるためにアクセスが必要な場合、\n マルチプレイを一度切断してアクセスし、\n ストーリーを進行させてから\n マルチプレイを再開してください。[b]",
-    "Text":	"Main Crew are not Availible\nduring multiplayer.\n\n[a 03]※If you need to advance the story,\ndisconnect from multiplayer to do so,\nthen restart multiplayer.[b]",
```

Comment:
This line was actually fine, the issue was due to the whitespace between "Text" and the text itself.